src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -0,0 +1,293 @@
+		};
+	},
+	computed: {
+		urlParams() {
+			return toUrlParams({
+				offset: this.offset,
+			});
+		},
+		loans() {
+			return _get(this.loanChannel, 'loans.values') || [];
+		},
+		totalCount() {
+			return _get(this.loanChannel, 'loans.totalCount') || 0;
+		},
+		loanQueryVars() {
+			return {
+				ids: [this.targetedLoanChannelID],
+				limit: this.limit,
+				offset: this.offset
+			};
+		},
+		filterUrl() {
+			// initial release sends us back to /lend
+			return `/lend/${this.$route.params.category || ''}`;
+		}
+	},
+	apollo: {
+		preFetch(config, client, args) {
+			return client.query({
+				query: loanChannelPageQuery
+			}).then(({ data }) => {
+				// filter routes on route.param.category to get current path
+				const targetedLoanChannelURL = _get(args, 'route.params.category');
+				// isolate targeted loan channel id
+				const targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, data);
+				// extract query
+				const pageQuery = _get(args, 'route.query');
+				// query our targeted loan channel
+				return client.query({
+					query: loanChannelQuery,
+					variables: _merge(
+						{
+							ids: [targetedLoanChannelID],
+							limit: loansPerPage
+						},
+						fromUrlParams(pageQuery)
+					)
+				});
+			}).then(() => {
+				// TODO: REMOVE Once Lend Increment Button EXP ENDS
+				// Pre-fetch the setting for lend increment button
+				return client.query({ query: experimentSetting, variables: { key: 'uiexp.lend_increment_button' } });
+			}).then(() => {
+				// TODO: REMOVE Once Lend Increment Button EXP ENDS
+				// Pre-fetch the assigned version for lend increment button
+				return client.query({ query: experimentQuery, variables: { id: 'lend_increment_button' } });
+			});
+		}
+	},
+	created() {
+		const allChannelsData = this.apollo.readQuery({
+			query: loanChannelPageQuery
+		});
+		// filter routes on param.category to get current path
+		const targetedLoanChannelURL = _get(this.$route, 'params.category');
+		// isolate targeted loan channel id
+		this.targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, allChannelsData);
+		// extract query
+		this.pageQuery = _get(this.$route, 'query');
+		// Read the page data from the cache
+		const baseData = this.apollo.readQuery({
+			query: loanChannelQuery,
+			variables: _merge(
+				this.loanQueryVars,
+				fromUrlParams(this.pageQuery)
+			)
+		});
+
+		this.isLoggedIn = !!_get(baseData, 'my');
+		this.isVisitor = !_get(baseData, 'my.userAccount.id');
+		this.itemsInBasket = _map(_get(baseData, 'shop.basket.items.values'), 'id');
+		this.loanChannel = _get(baseData, 'lend.loanChannelsById[0]');
+
+		// this.apollo.watchQuery({
+		// 	query: loanChannelQuery,
+		// 	variables: _merge(
+		// 		this.loanQueryVars,
+		// 		fromUrlParams(this.pageQuery)
+		// 	),
+		// 	fetchPolicy: 'network-only',
+		// }).subscribe({
+		// 	next: ({ data }) => {
+		// 		this.loanChannel = _get(data, 'lend.loanChannelsById[0]');
+		// 	},
+		// });
```

Comment:
This watchQuery is how things were done in the original lend-vue page. However, I couldn't get that work with these queries so I moved to an actual "watch" setup against the query vars below. @emuvente I'm open to revisiting this if there is a better way. Otherwise, I'll remove after you verify.